vertx-web-client/src/main/java/io/vertx/ext/web/client/HttpResponse.java

```diff
@@ -119,20 +120,29 @@
    */
   @CacheReturn
   @Nullable
-  String bodyAsString();
+  default String bodyAsString() {
```

Comment:
bodyAsJsonArray needs to access the buffer I think. this was done by @pavolloffay